### DISCUSSION TITLE
Using `continue_on_failure`

### DISCUSSION THREAD

**Discussion by sh1shir** [Gusto/buildkite-builder#119] [General]
Just wondering what the syntax looks like to use the `continue_on_failure` functionality for a `wait` looks like. In a typical `pipeline.yaml` it looks like this 

`steps:
  - command: "command.sh"
  - wait: ~
    continue_on_failure: true
  - command: "echo This runs regardless of the success or failure"
  - wait: ~
  - command: "echo The command passed"
`

**Comment by ngan** [comment]
You can do `wait(continue_on_failure: true)`. It’s sort of an odd ball not I figured it makes the wait directive simpler.

**Reply by sh1shir** [reply]
Thanks! Could you possibly document the syntax for some of the optional attributes for the step commands?

**Reply by ngan** [reply]
We don't document it because it's already documented on Buildkite's side:
https://buildkite.com/docs/pipelines/command-step

If there's anything missing, let us know!

**Comment by crossterra-agent** [comment]
Waiting for developer to complete implementation on PR https://github.com/CrossTerra/monorepo/pull/2214.
